### PR TITLE
Add support for ELRS UID

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: https://github.com/betaflight/betaflight-configurator/pull/2721

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: gulpfile.js
Comment (freasy) [inline gulpfile.js:1106]: if you leave this on, it will increase built size significant. I would turn sourmaps off for production.

File: src/js/tabs/receiver.js
Comment (freasy) [inline src/js/tabs/receiver.js:29]: to prevent a possible memory leak, move this line up to line #11.
This is still "local" but for sure only gets called once (receiver.init).

File: src/js/tabs/receiver.js
Comment (freasy) [inline src/js/tabs/receiver.js:367]: remove dev console logs

File: src/js/tabs/receiver.js
Comment (freasy) [inline src/js/tabs/receiver.js:369]: this complete block needs 1 intend tab

**Review by freasy — COMMENTED** [review]
Code LQGTM

File: src/js/tabs/receiver.js
Comment (haslinghuis) [reply src/js/tabs/receiver.js]: Thanks for the review. Should be fixed in last push.

**Review by jeffpearce — APPROVED** [review]

**Comment by nerdCopter** [comment]
* The UI & firmware setting worked for me.
* I  did not test bind due to not being ready to flash my module to ELRS 3.

**Comment by nerdCopter** [comment]
may need some css to make text-field look same as others.

**Comment by haslinghuis** [comment]
Here is my attempt:

image

**Review by blckmn — APPROVED** [review]

**Comment by nerdCopter** [comment]
looks prefect....  do you have light-mode done too? i dont feel like reflashing my whoop today :D

**Review by sugaarK — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
